### ISSUE TITLE
Duplicates of keywords

### ISSUE THREAD

**Issue by jennur** [zenodo/zenodo#2333]
Browser & OS (see also https://www.whatismybrowser.com/): <Browser>:<OS>

## Describe the bug
The "Taxonomy" keyword is listed twice in the sidebar, might also be the case with other keywords (?), as there seems to be a distinction between uppercase/lowercase versions. 

## Steps to Reproduce
1. Go to https://zenodo.org/
2. Search for "Taxonomy"
3. Scroll down to 'Keywords' in the sidebar
4. See "Taxonomy" listed twice
5. Inspect both elements and see the difference in capitalization.

## Expected behavior
All keywords in the sidebar are listed only once.

## Screenshots (if applicable)
<img width="255" alt="Screenshot 2022-06-01 at 15 26 14" src="https://user-images.githubusercontent.com/21052053/171415804-8e058b8a-4e24-48e2-9e58-af26c89b490b.png">
<img width="315" alt="Screenshot 2022-06-01 at 15 25 09" src="https://user-images.githubusercontent.com/21052053/171415783-36d636e7-9ec7-4c67-91ed-fff2745a131c.png">
<img width="295" alt="Screenshot 2022-06-01 at 15 25 17" src="https://user-images.githubusercontent.com/21052053/171415794-7bf9f4d3-8240-4667-ae12-210ed82ea84b.png">
<img width="387" alt="Screenshot 2022-06-01 at 15 25 24" src="https://user-images.githubusercontent.com/21052053/171415800-450fa1cf-fa86-45d2-8577-71121f3e6a4e.png">
